pkg/metrics/metrics.jsx

```diff
@@ -641,6 +646,39 @@ class CurrentMetrics extends React.Component {
             cpu_label = this.state.cpuUsed + '%';
         }
 
+        let disksUsage = [];
+        if (this.disksNames.length > 0 && this.samples[14] && this.samples[15]) {
+            disksUsage = this.disksNames.map((name, i) => [
+                name,
+                this.samples[14][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[14][i]) : "0",
+                this.samples[15][i] >= 1 ? cockpit.format_bytes_per_sec(this.samples[15][i]) : "0",
+            ]);
+            disksUsage = disksUsage.filter(disk => !disk[0].match(/^zram[0-9]+$/));
```

Comment:
This "zram" special-case seems odd. I agree that we don't want to show disk usage on swap partitions, LVM PVs and similar "not quite a file system" partitions. But this should be done more generally -- perhaps a positive check that you only show mounted partitions? Or at least filter out all swaps?

Reply:
Sorry, I walked into the same trap as above -- we *are* showing devices here, *not* file systems. So this comment was mostly bogus. It's still not obvious to me what we actually want to show here -- only physical disks? So not ram disks, scsi_debug, or things like iSCSI and NBD? That would require some definitions first.

Given that RAM disk usage is still usage (in terms of CPU and "why is my system busy" at least), what makes them special?

But I agree, we don't need to pinpoint this right now -- we can start with showing mostly everything, and cut down once we have some more in-the-field experience. So I'd actually prefer to drop this filter entirely for now, but I don't veto the zram special case either.

Reply:
I agree, let's not filter at all. Thanks for the input!